crates/primitives/receipt/src/from_blockifier.rs

```diff
@@ -75,7 +75,7 @@ pub fn from_blockifier_execution_info(res: &TransactionExecutionInfo, tx: &Trans
         Transaction::L1HandlerTransaction(tx) => match get_l1_handler_message_hash(tx) {
             Ok(hash) => Some(hash),
             Err(err) => {
-                log::error!("Error getting l1 handler message hash: {:?}", err);
+                tracing::error!("Error getting l1 handler message hash: {:?}", err);
```

Comment:
Consider instrumenting the parent function `from_blockifier_execution_info`

Reply:
this function should not just display a log but should report the error imo

Reply:
the error should be bubbled up right?

Reply:
yes, this comes from PR #321